src/earn/PoolCard.tsx

```diff
@@ -195,6 +156,7 @@ const styles = StyleSheet.create({
     marginBottom: Spacing.Thick24,
```

Comment:
This style should likely go as a gap in the parent component that contains the pool cards and not on the bottom of each pool card. Applying a margin to the bottom of each pool card causes an issue with the Android ripple with the bottom corners.

```suggestion
```

Reply:
It looks like adding a `gap: Spacing.Regular16` in [sectionListContentContainer of src/earn/PoolList.tsx](https://github.com/valora-inc/wallet/blob/main/src/earn/PoolList.tsx#L61) would keep this spacing and avoid coding the layout spacing into the card component.

Reply:
cleaned up in [0ced40b](https://github.com/valora-inc/wallet/pull/5787/commits/0ced40bb7aebc8de50b2af82141145ff9bf64ef2)